lambda-code/nagware/lib/overdueResponseCache.ts

```diff
@@ -0,0 +1,24 @@
+import { RedisConnector } from "./redisConnector.js";
+
+/**
+ * Handles caching of overdue form responses.  The cache expiry should be at least
+ * as long as the interval between Nagware function runs.
+ * 
+ * This cache is used by the app to reduce the number of DynamoDB calls required
+ * when loading the user's `/forms` page. 
+ */
+
+const OVERDUE_CACHE_EXPIRE_SECONDS = process.env.OVERDUE_CACHE_EXPIRE_SECONDS ?? "259200"; // 3 day default
+const OVERDUE_CACHE_KEY = process.env.OVERDUE_CACHE_KEY ?? "overdue:responses:template-ids"
```

Comment:
It looks like `process.env.OVERDUE_CACHE_KEY` is not set. We could just delete it and use the default value.

Reply:
Same thing here, we can just hard code it ;)